agents-and-function-calling/bedrock-agents/features-examples/13-create-agent-using-CDK/BedrockAgentStack/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "s3BucketName": "booking-agent-us-east1-account",
```

Comment:
shouldn't have the region hardcoded here.. can we add region and account name to bucket name in the code? and call this one bucket prefix

Reply:
100% agree, changed